### PR TITLE
fix #80755 a2zapk.com

### LINKED ISSUE #80755: a2zapk.com
### Issue URL (Anti Adblock Script)
 [https://a2zapk.com/1038364-sai-split-apks-installer-4-5-modded-a2z.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fa2zapk.com%2F1038364-sai-split-apks-installer-4-5-modded-a2z.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 34
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (30), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German
Custom filters: | https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt,<br>https://raw.githubusercontent.com/DandelionSprout/adfilt/master/LegitimateURLShortener.txt
Userscripts: | https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#80755

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -574,6 +574,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80755
+||pagead2.googlesyndication.com/getconfig/sodar$xmlhttprequest,redirect=nooptext,domain=jaanmusic.in
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=jaanmusic.in
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:579]: added a bit different, with redirect for adsbygoogle.js Current solution is not working at the moment.

**Comment by wout-codes** [comment]
@AdamWr

**Comment by wout-codes** [comment]
I can still reproduce it. It's fixed if I use my rules.

**Comment by Alex-302** [comment]
image

Please show screenshot. Do you say about page like on my screenshot?

**Comment by wout-codes** [comment]
Don't you read the what the reporter has described? Now I understand, why AdGuard is corrupted.

**Comment by AdamWr** [comment]
I guess this issue is about anti-adblock after clicking on `Direct Download APK From A2ZAPK` button.
By the way, this website is using the same anti-adblock method/script like dropgalaxy.

**Comment by wout-codes** [comment]
You can fix it using normal rules. Don't need that giant rule.

**Pull request closed without merging**